index.html

```diff
@@ -287,7 +287,8 @@ <h4 class="uppercase mb16">Lightbox Grid Gallery</h4>
 		        <div class="container">
 		            <div class="row mb64 mb-xs-24">
 		                <div class="col-sm-12 text-center spread-children-large">
-		                    <img alt="Pic" src="img/avatar1.png">
+			            <div class="col-2">
```

Comment:
Please close this tag. Also, I am not entirely sure you were wanting do `col-2` because you didn't add any css. Also given the styling of the website right now, you don't even need this <div> according to me. Correct me if i am wrong.

Also, in case of a frontend change either attach a screenshot or a link to the website using gh-pages. Helps to review for the mentor.

Reply:
@SaptakS , adding css or a bootstrap class was suggested as her image was covering a major part of the section